src/ProjectTemplates/Web.ItemTemplates/content/MvcController/.template.config/localize/templatestrings.en.json

```diff
@@ -0,0 +1,9 @@
+{
+  "author": "Microsoft",
+  "name": "MVC Controller",
+  "description": "A MVC Controller with or without read/write actions",
```

Comment:
Drop the leading "A" to make it `MVC Controller with or without read/write actions`

Reply:
sg ty